handler.js

```diff
@@ -1,18 +1,24 @@
 #!/usr/bin/env node
 // Executor of 'jobs' using the Redis task status notification mechanism
 
-const tracer = process.env.HF_VAR_ENABLE_TRACING  === "1" ? require("./tracing.js")("hyperflow-job-executor"): undefined;
-const { spawn } = require('child_process');
+const tracer = process.env.HF_VAR_ENABLE_TRACING === "1" ? require("./tracing.js")("hyperflow-job-executor") : undefined;
+const otelLogger = process.env.HF_VAR_ENABLE_TRACING === "1" ? require("./logs.js")("hyperflow-job-executor") : undefined;
```

Comment:
Here test a different env, e.g. `"HF_VAR_ENABLE_OTEL".` TRACING should be enabled separately because it requires proper context propagation.